mobile/native/src/event/api.rs

```diff
@@ -30,6 +30,7 @@ pub enum Event {
     BackgroundNotification(BackgroundTask),
     Authenticated(TenTenOneConfig),
     DlcChannelEvent(DlcChannel),
+    FundingChannelNotification(FundingChannelTask),
```

Comment:
🤔 do we really need a different kind of notification here? Why not use a background task?

Looks to me that's exactly what it is.

Reply:
I didn't pick the `BackgroundTask` to keep things separated.

Reply:
Hmm.. can you explain? From what do you want to keep it separate?

Reply:
From the flow how the `BackgroundTask` is being handled as I think I like to keep things separated. 
Imho, everything every event is a _background task_ as it's executed asynchronously in the _background_. 

Anyways, I have no hard feelings around this (💔). If you think I should merge it into the `BackgroundTask`, I'm happy to do so.

Reply:
No need, we can still change it, if we think it gets to annoying to maintain.